administration/installation-and-deployment/deployment-example.md

```diff
@@ -82,21 +82,12 @@ The initialization steps are also performed as `root` for the nodes that make up
   Each HA cluster node has its own unique SSH host keys stored under `${NCS_CONFIG_DIR}/ssh_host_ed25519_key`. The SSH client(s), here the manager, has the keys for all nodes in the cluster paired with the node's hostname and the VIP address in its `/root/.ssh/known_hosts` file.\
   \
   The host keys, like those used for client authentication, are generated each time the HA cluster nodes are initialized. The host keys are distributed to the manager and nodes in the HA cluster before the NSO built-in SSH and OpenSSH servers are started on the nodes.
-* As NSO runs in containers, the environment variables are set to point to the system install directories in the Docker Compose `.env` file.\
-  \
-  NSO runs as the non-root `admin` user and, therefore, the `ncs` command is used to start NSO instead of the `/etc/init.d/ncs` and `/etc/profile.d` scripts. The environment variables are copied to a `.pam_environment` file so that the `root` and `admin` users can set the required environment variables when those users access the shell via SSH.
-* The start script is installed as part of the NSO system install, and it can be customized if you would like to use it to start NSO. The available NSO start script variants can be found under `/opt/ncs/current/src/ncs/package-skeletons/etc`. The scripts may provide what you need and can be used as a starting point.
-*   If you are running NSO as the root user and using `systemd`, the `init.d` script can converted for use with `systemd`. Example:
-
-    ```bash
-    $ mkdir -p /etc/init.d
-    $ ./nso-NSO_VERSION.linux.x86_64.installer.bin --system-install
-    $ cp /etc/init.d/ncs /etc/rc.d/init.d/
-    $ systemctl daemon-reload
-    $ echo "ExecReload=/etc/rc.d/init.d/ncs reload" << /run/systemd/generator.late/ncs.service
-    $ cp /run/systemd/generator.late/ncs.service /etc/systemd/system/
-    $ systemctl start ncs
-    ```
+* As NSO runs in containers, the environment variables are set to point to the system install directories in the Docker Compose `.env` file.
+* NSO runs as the non-root `admin` user and, therefore, the NSO system installation is done using the `./nso-${VERSION}.linux.${ARCH}.installer.bin --system-install --run-as-user admin` options. The NSO installation start script will create a systemd system service to run NSO as the `admin` user (default is the `root` user) when NSO is started using the `systemctl start ncs` command. \
+\
+The environment variables are copied to a `.pam_environment` file so the `root` and `admin` users can set the required environment variables when those users access the shell via SSH.\
+\
+The `/etc/systemd/system/ncs.service` systemd service script is installed as part of the NSO system install, and it can be customized if you would like to use it to start NSO. The script may provide what you need and can be a starting point.
```

Comment:
```suggestion
The `/etc/systemd/system/ncs.service` `systemd` service script is installed as part of the NSO system install, and it can be customized if you would like to use it to start NSO. The script may provide what you need and can be a starting point.
```